index.html

```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Worrior' Cars</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="headerDesktop">
+      <div class="links">
+        <ul>
+          <li> <a href="https://web.facebook.com/?_rdc=1&_rdr"><img class="facebook-icon" src="Icons/facebook.png" alt="" /></a> </li>
+          <li> <a href="https://twitter.com/home"><img class="twitter-icon" src="Icons/twitter.png" alt="" /></a></li>
+          <li>English</li>
+          <li>MyPage</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="headerSecondary">
+          <nav>
+            <div>
+                <img id="mobileMenu" src="Icons/mobile-menu.png" alt="">
+                <img id="Mobilecross" src="Icons/cross.png" alt="">
+            </div>
+          <span class="menu">
+            <img src="icons/warrior.png" alt="" />
+            <ul>
+              <li id="about-item" class="about-item" >About</li>
+              <li>Program</li>
+              <li>Join</li>
+              <li>Sponsor</li>
+              <li>News</li>
+            </ul>
+            <button>Compaign</button>
+          </span>
+        </nav>
+      </div>
+    </header>
+
+    <div id="PopupMobileMenu">
+      <ul>
+        <div id="X">X</div>
+        <li id="about-item-mob" class="about-item" >About</li>
+        <li>Program</li>
+        <li>Join</li>
+        <li>Sponsor</li>
+        <li>News</li>
+      </ul>
+    </div>
+
+<main class="desktop-main" id="desktop-main">
+    
+    <section class="landing-page">
+        <div>
+            <h3>"The New Release!"</h3>
+            <h1>
+                Announcing Our <br />
+                Jaguare E-3 <br />
+                2023
+            </h1>
+            <p>
+                Worrior has released its latest electric car, the e-type 3.This
+                vehicle boasts a sleek design,improved driving performance, and a
+                longer battery range.
+            </p>
+            <h2 id="landing-pageH2">2023.2.7(MON) ~ 12(SAT)</h2>
+            <h4 id="landing-pageH4">@ F-9 Park, Panjab Gate, Islamabad</h4>
+        </div>
+    </section>
```

Comment:
Kindly ensure to use the right header hierarchy throughout your project😉 `h1` should be used as the main or introductory header and this should be the only `h1`  while other headers under the principal header can use the lower hierarchy (h2, h3, h4... etc)  on the page😉 other sections should then start with an `h2`  since they are under the main header and then, other headers under the section header should use the lower hierarchy headers😉

Kindly note that the sizes of these headers are not for styling purposes as you can style any header to your taste😉 But their principal use is to represent the hierarchy of headings😉 This will increase your accessibility score, allow screen readers to navigate through your project easily😊

Kindly make the necessary adjustment all throughout your project, as this has been misused in many places😉😉👍